components/SidebarNav/react.stories.js

```diff
@@ -1,149 +1,183 @@
 import React from 'react'
 import { storiesOf } from '@storybook/react'
-import { withKnobs } from '@storybook/addon-knobs'
+import { withKnobs, boolean } from '@storybook/addon-knobs'
 import {
-  mEntry, mInterior, mExport, mContact, mSignout,
+  mEntry, mExport, mContact, mSignout,
 } from '@masonite/svg-icons'
 import logo from 'images/masonite-icon.png'
 import SidebarNavLayout from './react'
 import Readme from './react.README.md'
 
 const addKey = item => ({ ...item, key: item.label })
 
-storiesOf('SidebarNav', module)
-  .addParameters({ readme: { sidebar: Readme } })
-  .addDecorator(withKnobs)
-  .add('Default', () => {
-    const onClick = () => console.log('Mock onClick call')
-    const location = {
-      pathname: '/glass/inventory',
-    }
-    const menuItems = [
-      {
-        label: 'Exterior Doors',
-        icon: <span dangerouslySetInnerHTML={{ __html: mEntry }} />,
-        active: true,
-        href: '/',
-        items: [
-          {
-            label: 'Menu Item 1',
-            href: '/menu/1',
-            onClick,
-          },
-          {
-            label: 'Menu Item 2',
-            href: '/menu/2',
-            onClick,
-          },
-        ].map(addKey),
-      },
+const combatStorybookPaddingStyle = {
+  position: 'fixed',
+  top: 0,
+  left: 0,
+}
+
+const Content = () => (
+  <div style={{ padding: 20 }}>
+    <h1>Page</h1>
```

Comment:
@tarikhamilton This h1 is valid for the _Default_ story however we would need this to be an h2 for the _Header Above Example_ story.

Reply:
Updating.